client/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 4,
```

Comment:
personal preference: can we keep it at 2 spaces? I find I have to do too much horizontal scrolling with 4 spaces.